vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -121,14 +123,19 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
           }
           vrrp_instance VI_1 {
               state MASTER
+{% if CAPM3_VERSION == "v1alpha3" %}
               interface eth1
+{% else %}
+              interface enp1s0
```

Comment:
@maelk is current state of PR meets the requirement of backwards compatibility and also is interface name `enp1s0` should be used here? I was not able to find the root cause of failure except this interface naming might be a problem..